flpproto.sh

```diff
@@ -4,12 +4,13 @@ requires:
   - Configuration
   - O2
   - "GCC-Toolchain:(?!osx)"
-build_requires:
+  - "PDA:(?!osx)"
```

Comment:
How would I restrict to CC7 only ? this is not restrictive enough I guess as it would still build PDA on Ubuntu.